job/job.go

```diff
@@ -0,0 +1,77 @@
+package job
+
+import (
+	gocontext "context"
+	"fmt"
+	"time"
+
+	"github.com/flanksource/duty/context"
+	"github.com/robfig/cron/v3"
+)
+
+type job struct {
+	context.Context
+	Name            string
+	Schedule        string
+	AllowConcurrent bool
+	Timeout         time.Duration
+	Fn              func(ctx JobRuntime) error
+	RunNow          bool
+}
+
+type JobRuntime struct {
+	context.Context
```

Comment:
I think this should be a field `Context context.Context` since we are passing `.Context` of this in all the funcs that use context.Context like db funcs